### PR TITLE
feat(integration-link): add --follow arg to manual-deploy command

### PR DESCRIPTION
**Pull request by gnuletik**
- [x] Add a changelog entry in the section "To Be Released" of CHANGELOG.md

Depends on https://github.com/Scalingo/go-scalingo/pull/332

### CODE REVIEW

**Comment by gnuletik** [comment]
I rebased my fork with `github.com/Scalingo/go-scalingo/v6@v6.5.0` @EtienneM

**Comment by gnuletik** [comment]
@EtienneM Let me know if anything is blocking this. Thanks!

File: cmd/integration_link.go [hunk 1]
```diff
@@ -356,9 +356,12 @@ List of available integrations:
...
 
 	integrationLinkManualDeployCommand = cli.Command{
-		Name:      "integration-link-manual-deploy",
-		Category:  "Integration Link",
-		Flags:     []cli.Flag{&appFlag},
+		Name:     "integration-link-manual-deploy",
+		Category: "Integration Link",
+		Flags: []cli.Flag{
+			&appFlag,
+			&cli.BoolFlag{Name: "follow", Usage: "Follow deployment"},
```
Comment (EtienneM) [inline cmd/integration_link.go:363]: let's also allow the short version of the flag
+ use the same wording as the `deployment-follow` command

```suggestion
			&cli.BoolFlag{Name: "follow", Aliases: []string{"f"}, Usage: "Follow deployment event stream"},
```

File: integrationlink/manual_deploy.go [hunk 2]
```diff
@@ -19,11 +20,24 @@ func ManualDeploy(ctx context.Context, app, branch string) error {
...
+	if !follow {
+		return nil
+	}
+
+	err = deployments.Stream(ctx, &deployments.StreamOpts{
+		AppName:      app,
+		DeploymentID: deploy.ID,
+	})
+	if err != nil {
+		return errgo.Mask(err, errgo.Any)
```
Comment (EtienneM) [inline integrationlink/manual_deploy.go:39]: convention: We no longer use `errgo.Mask`
```suggestion
		return errgo.Notef(err, "stream deployment logs")
```

**Review by EtienneM — CHANGES_REQUESTED** [review]
Sorry I didn't took the time earlier to review this PR

**Review by EtienneM — APPROVED** [review]

**Pull request merged**